pkg/utils/utils.go

```diff
@@ -186,3 +186,7 @@ func ParseNetworkID(networkID string) (string, string, error) {
 func GenerateNetworkID(network *v1.NetworkSelectionElement) string {
 	return fmt.Sprintf("%s_%s", network.Namespace, network.Name)
 }
+
+func GeneratePodNetworkID(pod *kapi.Pod, networkID string) string {
+	return string(pod.UID) + "_" + networkID
```

Comment:
nit: this could have been implemented the same as the above function for uniformity, but its fine :)
perhaps with a common internal function `func appendWithToken()` anyway this is not needed.